test/shared/src/test/scala/zio/test/TestAspectSpec.scala

```diff
@@ -124,15 +124,15 @@ object TestAspectSpec extends DefaultRuntime {
   def failureDoesNotMakesTestsPassOnUnexpectedAssertionFailure: Future[Boolean] =
     unsafeRunToFuture {
       val spec = test("test")(assert(true, equalTo(false))) @@ failure(
-        isCase[TestFailure[Boolean], TestResult](
```

Comment:
This one should be `TestResult` because we want the type safety of making sure the thing we're checking for equality against is also a `TestResult`, versus the one above the assertion was `anything` which just takes an `Any`. Otherwise all good.